labman/gui/templates/plate.html

```diff
@@ -115,37 +125,42 @@
   </select>
 </div>
 
-<!-- Studies div -->
-<div>
-  <label><h4>Studies being plated</h4></label>
-  <button class='btn btn-success' data-toggle='modal' data-target='#addStudyModal'><span class='glyphicon glyphicon-plus'></span> Add study</button>
-  <div id='study-list'>
-  </div>
-</div>
+<!--Only enable this after plate configuration has been selected-->
+<button id='createPlateBtn' class='btn btn-success' onclick='createPlate();' disabled>Create</button>
 
-<!-- Plate map div -->
-<h4>Plate map</h4>
-
-<!-- Controls div -->
-<div><h5>Controls description</h5>
-  <p>Press <kbd class='key'>ESC</kbd> to enter cell <i>selection mode</i></p>
-  <p>Press <kbd class='key'>ENTER</kbd> or double click a cell to enter <i>edit mode</i></p>
-  <p>To add new Control types, go to <a target='_blank' href="/sample/manage_controls">Admin > Manage control sample types</a></p>
-    <table>
-      {% for cd in controls_description %}
-        <tr>
-          <td>{{cd['external_id']}}:</td>
-          <td>{{cd['description']}}</td>
-        </tr>
-      {% end %}
-    </table>
-  </span>
-</div>
+<div id="plateDetails" hidden>
+    <!-- Studies div -->
+    <div>
+      <label><h4>Studies being plated</h4></label>
+      <button class='btn btn-success' data-toggle='modal' data-target='#addStudyModal'><span class='glyphicon glyphicon-plus'></span> Add study</button>
+      <div id='study-list'>
+      </div>
+    </div>
 
-<div id='plate-map-div' class='spreadsheet-container'></div>
+    <!-- Plate map div -->
+    <h4>Plate map</h4>
 
-<h4>Well comments</h4>
-<div id='well-plate-comments' class="comment-box"></div>
+    <!-- Controls div -->
+    <div><h5>Controls description</h5>
+      <p>Press <kbd class='key'>ESC</kbd> to enter cell <i>selection mode</i></p>
+      <p>Press <kbd class='key'>ENTER</kbd> or double click a cell to enter <i>edit mode</i></p>
```

Comment:
Worth adding the instruction to right-click on a well to add comments to that well? Otherwise hard to discover.